DATA/production/qc-workflow.sh

```diff
@@ -176,6 +176,13 @@ elif [[ -z ${QC_JSON_FROM_OUTSIDE:-} ]]; then
         add_QC_JSON MCH_TRACKS $O2DPG_ROOT/DATA/production/qc-async/mch-tracks.json
       fi
     fi
+    if has_detectors_reco MFT MCH MID && has_detector_matching MFTMCH && has_detector_matching MCHMID; then
+        add_QC_JSON MFTMCHMID_TRACKS $O2DPG_ROOT/DATA/production/qc-async/mftmchmid-tracks.json
+    elif has_detectors_reco MFT MCH && has_detector_matching MFTMCH; then
+        add_QC_JSON MFTMCH_TRACKS $O2DPG_ROOT/DATA/production/qc-async/mftmch-tracks.json
+    elif has_detectors_reco MCH MID && has_detector_matching MCHMID; then
+        add_QC_JSON MCHMID_TRACKS $O2DPG_ROOT/DATA/production/qc-async/mchmid-tracks.json
+    fi
```

Comment:
Hi @aferrero2707 , I am wondering whether "has_detector_matching" should not be replaced by "has_matching_qc", to add the possibility to (de)activate these QC tasks?

Reply:
@pillot good point, thanks!